hoodie-client/src/main/java/com/uber/hoodie/io/HoodieCommitArchiveLog.java

```diff
@@ -195,7 +197,9 @@ public void archive(List<HoodieInstant> instants) throws HoodieCommitException {
             for (HoodieInstant hoodieInstant : instants) {
                 records.add(convertToAvroRecord(commitTimeline, hoodieInstant));
             }
-            HoodieAvroDataBlock block = new HoodieAvroDataBlock(records, wrapperSchema);
+            List<HoodieLogBlockMetadata> metadata = new ArrayList<>();
+            metadata.add(new HoodieInstantTime("100"));
```

Comment:
reading out of context.. whats special about `100`

Reply:
Oops, it's a typo, will be more careful.